docker/stellargraph-neo4j/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM neo4j:3.5
```

Comment:
Do we want to build this docker image in `stellar-devops` and push our own neon4j image and, or keep it here? You mentioned building the image is fast, but it'd offload the step

Reply:
I think it would be better to keep it here: it's not currently published anywhere, just built transiently, and, even if we did push it (which we could do with prebuilding and caching here), it's seems slightly nicer for the original Dockerfile be part of the open source library, rather than just published publicly from a private repo. What do you think?

(We can always change our approach in future, too.)

Reply:
Yeah sounds like a good plan 👍